File: Data Structures & Algorithms/number-of-senior-citizens/submission-1.py
class Solution:
    def countSeniors(self, details: List[str]) -> int:
        ans=0
        for i in range (len(details)):
            print(details[i][11]+details[i][12])
            if(int(details[i][11]+details[i][12])>60):
                ans+=1
        return ans
        

File: Data Structures & Algorithms/string-matching-in-an-array/submission-1.cpp
class Solution {
public:
    vector<string> stringMatching(vector<string>& words) {
        int n=words.size();
        vector<string> ans;
        unordered_set<string> st;
        sort(words.begin(),words.end(),
            [](string & a,string & b){
                return a.size()>b.size();
            });
        for(string ch:words) cout<<ch<<" ";
        for(int i=0;i<n;i++){
            for(int j=i+1;j<n;j++){
                if(words[i].find(words[j])!=string::npos) st.insert(words[j]);
            }
        }
        for(auto x:st){
            ans.push_back(x);
        }
        return ans;
    }
};